store/dummy_message_store.go

```diff
@@ -158,3 +175,7 @@ func (dms *DummyMessageStore) startSequenceSync() {
 	}
 	dms.stoppedC <- true
 }
+
+func (dms *DummyMessageStore) Check() error {
```

Comment:
I don't think it's necessary to implement a Check that does nothing. I removed the Check from both interfaces in store. Only if the implementations have a Check(), it will be used by the router when setting  its health-status.